src/core/components/ui/WalletBinding/index.vue

```diff
@@ -0,0 +1,191 @@
+            u Learn How to Connect ?
+
+          div(class="mb-5 ml-8 mr-8 mt-8")
+            v-btn(
+              dark
+              class="mt-5 align-center"
+              outlined
+              width="100%"
+              large
+              elevation="2"
+              color="secondary"
+              @click="setWalllet('metamask')"
+            ) Connect
+              
+        div(class="mt-10 mb-10 ml-10 mr-10" v-if="putAccount")
+        div
+          div(class="align-center mb-5") Your address
+          div.address( class="dg-card pb-2 pt-2", style="background: #eeeeee", elevation="0")
+            div(class="ml-3 p4") {{accountList[0].address}}
+              v-btn(icon @click="copyToClipboard(accountList[0].address)")
+                v-icon mdi-vector-arrange-below
+            
+          v-row(class="mt-5")
+            v-col DAI Balance
+            v-col(cols="auto") {{accountList[0].balance}} ETH
+              
+          v-row
+            v-col
+            v-col(cols="auto") ({{accountList[0].balance}} USD)
+          
+          v-btn(
+            class="mt-5 align-center"
+            outlined
+            color="secondary"
+            width="100%"
+            large
+            light
+          ) Disconnect Wallet            
+</template>
+
+<script>
+import { mapState, mapMutations } from "vuex";
+import { startApp } from "@/common/lib/metamask";
+import { getBalanceETH } from "@/common/lib/metamask/wallet.js";
+import { setEthAddress } from "@/common/lib/polkadotProvider/command/userProfile";
+import { getEthFromFaucet, getDaicFromFaucet } from "@/common/lib/faucet";
+
+export default {
+  name: "WalletBinding",
+  props: {
+    show: Boolean,
+  },
+  data: () => ({
+    isLoading: false,
+    error: "",
+    putWallet: true,
+    putAccount: false,
+    loading: false,
+    ethAccount: null,
+    accountList: [],
+    inputPassword: false,
+    selectAccount: null,
+  }),
+  computed: {
+    _show: {
+      get() {
+        return this.show;
+      },
+      set(val) {
+        this.$emit("toggle", val);
+      },
+    },
+    ...mapState({
+      api: (state) => state.substrate.api,
+      wallet: (state) => state.substrate.wallet,
+      metamaskWalletAddress: (state) => state.metamask.metamaskWalletAddress,
+    }),
+  },
+  watch: {
+    show() {
+      if (this.show) {
+        this.error = "";
+        this.loading = false;
+        this.putAccount = false;
+        this.putWallet = true;
+        this.inputPassword = false;
+        this.isLoading = false;
+      }
+    },
+  },
+  mounted() {},
+  methods: {
+    ...mapMutations({
+      setMetamaskAddress: "metamask/SET_WALLET_ADDRESS",
+    }),
+    async getMunnyFromFaucet(address) {
+      await getEthFromFaucet(address);
+      await getDaicFromFaucet(address);
+    },
+    async setWalllet(walletName) {
+      this.error = "";
+      this.loading = true;
+      switch (walletName) {
+        case "metamask":
```

Comment:
Do not use, switch case, please separate this into a separate file if possible. Inside the `common\lib\metamask` directory. You can discuss this with me on Discord for the details.

Reply:
@agustinustheo changed as per requested, kindly re-review ya mas . thank you :)